DESCRIPTION

```diff
@@ -1,6 +1,5 @@
 Package: LPWC
-Version: 0.99.5
-Date: 2019-03-12
```

Comment:
Should we keep the date?

Reply:
No we do not need to keep the date. Usually, CRAN overwrites it.

Reply:
In the past, we usually submit and add the date back.